lib/java/deploy/MavenPerBranchSpringBootDeploymentGoal.ts

```diff
@@ -34,6 +35,31 @@ import * as os from "os";
 import * as portfinder from "portfinder";
 import { MavenLogInterpreter } from "../../maven/build/mavenLogInterpreter";
 
+export const ListBranchDeploys: CommandHandlerRegistration = {
+    name: "listLocalDeploys",
+    intent: "list branch deploys",
+    description: "list branch deploys",
```

Comment:
Perhaps something more prose-y and explanatory, e.g., "List deployments of repository across all branches".

Reply:
Done